src/services/billing-services/list-service-and-products-changes-accounting-service.js

```diff
@@ -244,9 +244,17 @@ const joinEdgeApplicationWithTieredCache = (services) => {
     }
   })
 
-  services.splice(tieredCacheServiceIndex, 1)
-  services.splice(edgeStorageServiceIndex, 1)
-  services.splice(botManagerServiceIndex, 1)
+  const indicesToRemove = [
```

Comment:
hum ... Can we add a test for this?
Either unit or something (on Cypress) like "Ensure the last product from the list is visible on screen". e2mock on this case.